craft_parts/packages/base.py

```diff
@@ -34,6 +34,8 @@
 class BaseRepository(abc.ABC):
     """Base implementation for a platform specific repository handler."""
 
+    stage_packages_filters: Optional[Iterable[str]] = None
```

Comment:
So just so I understand how the tools will set this class attribute: will rockcraft (for example), set this value based on the project's `base` at runtime?